examples/tic-tac-toe/move/Move.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "tic_tac_toe"
+edition = "2024.beta"
+
+[dependencies]
+Sui = { local = "../../../crates/sui-framework/packages/sui-framework" }
```

Comment:
Should we depend on the local framework here? I remember on the trading demo, we did depend on a live sui branch (testnet iirc) to make these easy to copy + play with outside of the monorepo. I think whichever route we choose, we should do it consistently!

Reply:
I think we should depend on the local one everywhere for now, so that CI doesn't have to refetch the sui repo. This problem should go away once we eliminate the sui framework as an explicit dependency.

Reply:
I can make the change to the trading contracts!

Reply:
https://github.com/MystenLabs/sui/pull/18573